doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -0,0 +1,84 @@
+How to Set Up MoveIt 2 Docker Containers in Ubuntu
+===================================================
+This guide will provide a walkthrough on how to get a Docker container with MoveIt 2 dependencies set up quickly.
+It includes a script that will get you up and running in MoveIt quickly!
+This guide is intended for people who would like to have a separate environment for working with MoveIt up and running quickly \
+without having to do much configuring. In this guide, we will be setting up a ROS2 Galactic environment.
```

Comment:
As this targets the main branch it would be really nice if this was for Ubuntu 22.04 and Rolling. If you make that switch this tutorial should also easily apply to Humble. Of course we can also backport this to Galactic as well.